config.example.yml

```diff
@@ -11,8 +11,10 @@ cpus: 2
 # - shortname: The site's shortname.
```

Comment:
Should we mention the `shared_folders` option in config? With sample syntax but just all commented out?